amanuensis/__init__.py

```diff
@@ -66,17 +64,10 @@ def app_init(
 
 
 def app_sessions(app):
+    ''' Override userdatamodel's `setup_db` since Alembic handles the migrations now. '''
     app.url_map.strict_slashes = False
+    SQLAlchemyDriver.setup_db = lambda _: None
```

Comment:
This _might_ need to be commented out/removed for the very first time prior to running Alembic for the first time. It just depends if the setup_db scripts ran at some point prior to this update. If they did, then it's fine. The work around here would be to check if the alembic_version table exists, but it would only need to be used once, so I'm not sure what's the best way to handle this.